.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    branches: [master]
+    paths: "**/Cargo.toml"
+
+jobs:
+  Publish:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1 # ensure crate order
+      fail-fast: false
+      matrix:
+        crate: [
+            'ndk-sys',
+            'ndk',
+            'ndk-glue',
+            'ndk-build',
+            'cargo-apk',
+        ]
+        rust-target: [ 'armv7-linux-androideabi' ]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        target: ${{ matrix.rust-target }}
+        override: true
+    - name: Publish ${{ matrix.crate }}
+      uses: actions-rs/cargo@v1
+      with:
+        command: publish
+        args: --manifest-path ${{ matrix.crate }}/Cargo.toml --dry-run --target ${{ matrix.rust-target }} --token ${{ secrets.cratesio_token }}
```

Comment:
Won't `--dry-run` keep this from actually publishing?

Also, I've personally been using `continue-on-error: true` to keep CI from turning red if Cargo.toml is modified but the version isn't bumped... but that's not a perfect solution, since it could obscure publish failures.

Reply:
oops! fixed